### DISCUSSION TITLE
data.in

### DISCUSSION THREAD

**Discussion by omarwater98** [gmtsar/gmtsar#434] [Q&A]
I made a data.in file and copy from the measurement folder the name of the subswath 3 with the polarization "vv" of the 10 images that I am going to use, each one with its orbit file as indicated by ":" the manual in the page 14, but when running the command:

preproc_batch_tops.csh data.in dem.grd 1 &> pbt_mode1.log &

nothing is generated in the baseline_table.dat file

**Comment by Xiaohua-Eric-Xu** [comment]
What's in your pbt_mode1.log?

**Reply by omarwater98** [reply]
hi, in the pbt_mode1.log was written  didnt has linked the data file name, i just resolved. in the page 13 5. Choosing Master Image the command
ln -s ../../data/F*/*.SAFE/*/*iw1*vv*xml .
i didnt have any folder called Fxxxx_Fxxxx so then deleted the 'F*' 

Thanks for the atention to answer me, i appreciate it
